section4/exercises/burrito.rb

```diff
@@ -11,9 +11,25 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(topping)
+    toppings.push(topping)
```

Comment:
Nice job with your dynamic methods here! Don't forget to communicate which toppings you are adding too... that is what the `@` does for you.